include/vkd3d_command_list_vkd3d_ext.idl

```diff
@@ -0,0 +1,32 @@
+/*
+ * * Copyright 2021 NVIDIA Corporation
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+import "vkd3d_d3d12.idl";
+
+typedef struct VkCommandBuffer_T *VkCommandBuffer;
+
+[
+    uuid(77a86b09-2bea-4801-b89a-37648e104af1),
+    object,
+    local,
+    pointer_default(unique)
+]
+interface ID3D12GraphicsCommandListExt : IUnknown
```

Comment:
Do we know that this is enough? Is the plan to add CommandListExt1 etc later if we realize something is missing?

Reply:
If you mean methods, surely we need to more here as needed. At this point we do not have list to be added.

Reply:
Right, but that means ABI breakage? I assume if more interfaces are required, we'd have to add new derived interfaces in the future.